lib/core/src/server/preview/iframe-webpack.config.js

```diff
@@ -159,8 +179,9 @@ export default async ({
       extensions: ['.mjs', '.js', '.jsx', '.ts', '.tsx', '.json', '.cjs'],
       modules: ['node_modules'].concat(raw.NODE_PATH || []),
       alias: {
-        'babel-runtime/core-js/object/assign': require.resolve('core-js/es/object/assign'),
-        semver: require.resolve('@storybook/semver'),
+        ...themingPaths,
+        ...storybookPaths,
+        semver: path.dirname(resolveFrom(__dirname, '@storybook/semver/package.json')),
```

Comment:
Do we still need this ? 
I think we did it for IE 11 ?

Reply:
Yeah we did that for IE11, no longer needed you think?

Reply:
Nop, it's being transpiled. We did that because we thought it was using the wrong reference in the preview.